### ISSUE TITLE
[PM, Research] My VA C&A: Discuss next steps with Chante

### ISSUE THREAD

**Issue by ajakabcin** [department-of-veterans-affairs/va.gov-team#80438]
## Background
Over the last few sprints we have been doing some discovery around the Claims Status Tool and our own Claims & Appeals section on My VA. Matt has been leading this effort and has documented some findings for discussion. We would like to meet to discuss these findings and determine next steps with possible enhancements to the C&A section on My VA as well as notifications as these two are closely related.

## Tasks
- [x] Schedule time with Chante to review and discuss findings
- [x] Discuss potential next steps 

## Acceptance Criteria
- [ ] Path forward with Claims and Appeals and/or notifications work

**Comment by mattmarino-adhoc** [comment]
Some next steps:
CST:
1. Begin planning/work on inclusion of new slim alert for Claims cards with evidence requests
2. Create concepts, specifications in Figma
3. Engineering team will implement

Badge indicators:
1. Review past efforts related to notification center, badge concepts on MHV
2. Create badge indicator concepts and specifications in Figma, as well as documenting specific use cases for rollout
3. Sync with MHV on how to work together/considerations with services being used

Other notifications:
1. Continue to look at and consider other ways to surface actionable and otherwise important info for users
2. Sync with Mobile team to discuss how we can achieve parity in messaging

Events, alerts, community and Vet organization news:
1. Holding off for now, but we will investigate more in the future

**Comment by mattmarino-adhoc** [comment]
My VA Proposed Next Steps
Following the dot indicator study and review of the Claims Status Tool (CST), we’ve compiled some recommendations for next steps and prioritization. These are next steps recommendations that we would break out into more detailed tasks/tickets and prioritize over the next several sprints.
Recommendations from Matt’s read-out:
1.	Badge indicators should be considered as an option for a new notification type to use for health care messaging and updates, and in turn added to the VA Design System so it can be consumed by other teams for additional items that need user attention.
2.	The slim alert for evidence requests on claim cards should be introduced to My VA, for the sake of better informing Veterans of when they need to act and provide more evidence/details, as well as consistency with the Claim Status Tool.
3.	Notifications that matter most for Veterans should be elevated more than just these methods - something high-level that is displayed immediately to users when they've been authenticated. This could be a persistent presence for users that lives in top-level navigation, appearing on all pages beyond just My VA. 
4.	Investigate how Veterans could benefit from regional, location-specific, and personalized VA events, news, and alerts being shown on My VA, as well as determine the measures needed to incorporate such an addition.

Recommendations from CST discovery:
1.	Stay in touch with Benefits team and work in tandem where possible but don’t exceed their own pace
2.	Form a new strategy for working with the Mobile team. This has been a significant challenge in the past. How can we build (re-build?) the bridge?
3.	Begin Event bus exploration (aligns with recommendation 3 above)
4.	Determine how we want to interact with Event Bus

Proposed Next Steps
1.	Badge Indicator - recall that the main reason we needed to do the dot indicator et al. study was to fuel the process of getting the dot (or any like solution) into the design system.
a.	Meet with the MHV team to share research findings and let them know that we are pursuing the badge for unread messages as well as the process for design system adoption.
b.	Proceed with collab cycle and implementation steps. 
c.	Re-open experimental design/design system process to get the badge component added to the system.
2.	Slim Alert Banner - this is already part of the experimental design process if not admitted to the design system already. As findings suggest, let’s mirror what the Benefits CST team is doing and add this to the claims and appeals section card specifically for “evidence requests.”
a.	Proceed with collab cycle and implementation steps to 
3.	Written/On-Site Notifications Strategy 
a.	Revisit/review why we abandoned the Notification Center work
b.	Begin Event Bus exploration 
i.	Read documentation 
ii.	Set up intro call(s) with that group
iii.	Determine if this is a viable path for us
4.	Community Events exploration - as Matt noted in findings and we’ve heard in other research efforts, Veterans are seeking opportunities to feel part of a community. How might Veterans benefit from regional, location-specific, and personalized VA events?
a.	Internal discovery - have other teams done any work on this? We may need to look outside of OCTO.
b.	Hold off on this for now
